.github/workflows/release.yml

```diff
@@ -19,6 +19,8 @@ jobs:
           with:
             php-version: '7.4'
             ini-values: memory_limit=2048M
+          env:
+            COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Please remove these, why would you need to modify our secrets to add monolog support? This is a huge security concern...

Reply:
This wasn't meant for optimization but only as a temporary change in order to be able to pull in my google/gax fork. Without it, I ran into permission errors, for example here: https://github.com/googleapis/google-cloud-php/runs/6853003115?check_suite_focus=true

My plan was to revert these changes once the fork isn't needed anymore 😅

Reply:
Can confirm it doesn't hurt to have this in your github actions. It just make it easier for contributors as they can test against forks. i keep it in every project to make things easier.

Reply:
In the meantime, the tokens have been removed 😅